examples/restaurant_demo/restaurant.py

```diff
@@ -78,18 +78,21 @@ def sum_revenue_per_day_with_preaggregation(self) -> tuple:
     def get_non_private_counts_per_hour(self) -> dict:
```

Comment:
Do you think we should have a docstring for this method?

Also, since PyDP is in early days, it may be best to start using triple quotes/a delimiter (""") instead of a hash (#) for all docstrings. This can potentially help with auto-generating (API) docs.